packages/tabs/src/TabButton/TabButton.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
```

Comment:
as discussed on Slack, we need to use the `yarn create:component -o` script to create component options (sub-components).

What you could do is check out the `CardHeader` component `packages/card/src/CardHeader` and update your files according to this template, but returning the proper markup for the button, content, item, and wrapper.